tests/framework/CodeInterfaceTests/CobraTF/CobraTFinterfaceTest1/testDummyStep_MC/1/case5-15.ctf.out

```diff
@@ -0,0 +1,1652 @@
+h********************************* input file listing *********************************
+      12345678901234567890123456789012345678901234567890123456789012345678901234567890
```

Comment:
are we OK with putting this input file in the repo?
are we OK from an export control point of view?